src/h264_subscriber.cpp

```diff
@@ -47,7 +48,8 @@ void H264Subscriber::internalCallback(const h264_image_transport_msgs::H264Packe
         ROS_ERROR("Could not decode H264 frame");
```

Comment:
This should probably be warning, as it is non-fatal.

Reply:
Done